chain/find_test.go

```diff
@@ -8,15 +8,12 @@ import (
 	"testing"
 	"time"
 
-	"github.com/filecoin-project/go-address"
```

Comment:
These changes should be removed and addressed separately in a different PR that fixes https://github.com/filecoin-project/lily/issues/767